launcher/src/backend/ValidatorAccountManager.js

```diff
@@ -33,7 +33,13 @@ export class ValidatorAccountManager {
       }
       return readFileSync(file.path, { encoding: "utf8" });
     });
-
+    //escape all passwords for shell
+    passwords = passwords.map((p) => {
+      let pass = p;
+      if (pass.includes('"')) pass = pass.replaceAll(/"/g, '\\"');
```

Comment:
Is it better to use a library to escape all potential special characters rather than manually escaping only some of them? For example: `require('shell-escape')`

Reply:
they're escaped by JSON.stringify. The main problem are the double and single quotes. And because JSON.stringify already does escaping i need to remove the extra `\` introduced with `'\''`. This was all more complex then i thought tbh 😅

Reply:
but i think there is another problem when the password includes `\`...